datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -644,7 +644,7 @@ def _metric_tags(self, metric_name, val, sample, scraper_config, hostname=None):
         for label_name, label_value in iteritems(sample[self.SAMPLE_LABELS]):
             if label_name not in scraper_config['exclude_labels']:
                 tag_name = scraper_config['labels_mapper'].get(label_name, label_name)
-                _tags.append('{}:{}'.format(tag_name, label_value))
+                _tags.append(u'{}:{}'.format(tag_name, label_value))
```

Comment:
Seems this will change will fail in py2 if someone pass something like this `'my_2nd_label_value_é'`:

```
def test_submit_gauge_with_labels(aggregator, mocked_prometheus_check, mocked_prometheus_scraper_config):
    """ submitting metrics that contain labels should result in tags on the gauge call """
    ref_gauge = GaugeMetricFamily(
        'process_virtual_memory_bytes',
        'Virtual memory size in bytes.',
        labels=['my_1st_label', 'my_2nd_label'],
    )
    ref_gauge.add_metric(['my_1st_label_value', 'my_2nd_label_value_é'], 54927360.0)

    check = mocked_prometheus_check
    metric_name = mocked_prometheus_scraper_config['metrics_mapper'][ref_gauge.name]
    check.submit_openmetric(metric_name, ref_gauge, mocked_prometheus_scraper_config)
    aggregator.assert_metric(
        'prometheus.process.vm.bytes',
        54927360.0,
        tags=['my_1st_label:my_1st_label_value', 'my_2nd_label:my_2nd_label_value_é'],
        count=1,
    )
```

Is there a way to know what was the initial failure case ? which integration failed ?

Reply:
This seems to work for both cases: `'my_2nd_label:my_2nd_label_value_é` and `u"lab\xc3\xa9l"`

```python
from __future__ import unicode_literals # top of the file
...
...
_tags.append('{}:{}'.format(ensure_unicode(tag_name), ensure_unicode(label_value)))
```